dist/index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet"/>
+    <title>To Do List</title>
+    <link rel="stylesheet" href="./style.css">
```

Comment:
- [ ] CSS must be loaded by Webpack Style/CSS Loader, which means that you should import it inside the javascript file, kindly implement this requirement, example `import '/style.css'` 😄 
    - You should not import it with the `<link/>` tag.

Reply:
This issue is well resolved! ✔️